syft/rekor/queryRekor/syft.go

```diff
@@ -0,0 +1,87 @@
+package queryRekor
```

Comment:
name of file probably should be something like query_rekor.go or queryrekor.go. Since syft is already encoded in the prefix of the package